Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -434,6 +434,9 @@
 ### MinBias fastsim_13 TeV for mixing ###                                                                                                                                      
 workflows[2017.8] = ['',['MinBiasFS_13_UP17_ForMixing']]
 
+### MinBias fastsim_13 TeV for mixing, 2018 ###                                                                                                                                      
+workflows[2018.8] = ['',['MinBiasFS_13_UP18_ForMixing']]
```

Comment:
I think we also need fastSim noPU workflows from 2018.1 to 2018.13

Reply:
added